test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -415,11 +415,11 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 		await driverHelper.waitTillNotPresent( this.driver, this.publishingSpinnerSelector );
 		await driverHelper.waitTillPresentAndDisplayed(
 			this.driver,
-			By.css( '.editor-post-publish-panel__header-published' )
+			By.css( '.post-publish-panel__postpublish-header' )
 		);
 		return await driverHelper.verifyTextPresent(
 			this.driver,
-			By.css( '.editor-post-publish-panel__header-published' ),
+			By.css( '.post-publish-panel__postpublish-header' ),
 			'Scheduled'
```

Comment:
This doesn't seem to be set anymore in 8.5.1:
| 8.3.0  | 8.5.1 |
| ------------- | ------------- |
| <img width="299" alt="Screenshot 2020-07-27 15 46 19" src="https://user-images.githubusercontent.com/1451471/88549547-b7d5e280-d020-11ea-8fe9-a78992d3cbc8.png"> | <img width="296" alt="Screenshot 2020-07-27 15 45 35" src="https://user-images.githubusercontent.com/1451471/88549316-6decfc80-d020-11ea-8dfb-c392d2d9d2f2.png"> |

Reply:
The updated selector matches this element. It seems to match "scheduled" in the text.

![Screen Shot 2020-07-27 at 19 20 57](https://user-images.githubusercontent.com/841763/88571725-60466f80-d03e-11ea-833e-f55d653e8a8e.png)

It seems like this should work well across versions, but let's see what the tests report.

Reply:
It was previously matching the header selector, but I think it's OK to keep it as a check for that summary text. I'd only make it lowercased in the test to avoid confusion.